ledger/participant-integration-api/src/main/scala/platform/indexer/StandaloneIndexerServer.scala

```diff
@@ -23,78 +23,17 @@ final class StandaloneIndexerServer(
 )(implicit materializer: Materializer, loggingContext: LoggingContext)
     extends ResourceOwner[ReportsHealth] {
```

Comment:
Instead of extending `ResourceOwner`, I suggest making an `owner` method (that accepts the above parameters) and returns `indexerMigrations.flatMap(_ => indexerServer)`.